locales/languages/en.json

```
@@ -1906,7 +1906,9 @@
     "onboarding": {
       "what_to_expect": "What to Expect",
       "quotes": "Our buy crypto feature aggregates quotes from integrated vendors, providing quotes from those sources to get crypto directly into your wallet with no waiting period.",
+      "quotes_sell": "Now you can cash out directly in MetaMask! Get up-to-the-minute quotes from trusted providers while we guide you every step of the way.",
```

```suggestion
      "quotes_sell": "You can now cash out directly in MetaMask! Get up-to-the-minute quotes from trusted providers while we guide you through every step of the way.",
```
Nit to flow a bit better

---

But feel free to address only if needed